src/generators/python/presets/Pydantic.ts

```diff
@@ -0,0 +1,32 @@
+import { PythonOptions } from '../PythonGenerator';
+import { ClassPresetType, PythonPreset } from '../PythonPreset';
+import { PYTHON_DEFAULT_ENUM_PRESET } from '../renderers/EnumRenderer';
+
+const PYTHON_PYDANTIC_CLASS_PRESET: ClassPresetType<PythonOptions> = {
+  async self({renderer, model}) {
+    renderer.addDependency('from typing import Optional, Any');
+    renderer.addDependency('from pydantic import BaseModel, Field');
+
+    const defaultClassString = await renderer.defaultSelf();
+
+    return defaultClassString.replace(`class ${model.name}:`, `class ${model.name}(BaseModel):`);
+  },
+  property(params) {
+    const type = params.property.required
+      ? params.property.property.type
+      : `Optional[${params.property.property.type}]`;
+    const alias = params.property.property.originalInput['description']
+      ? `alias='${params.property.property.originalInput['description']}'`
+      : '';
+
+    return `${params.property.propertyName}: ${type} = Field(${alias})`;
+  },
+  ctor: () => '',
+  getter: () => '',
+  setter: () => ''
+};
+
+export const PYTHON_PYDANTIC_PRESET: PythonPreset<PythonOptions> = {
+  class: PYTHON_PYDANTIC_CLASS_PRESET,
+  enum: PYTHON_DEFAULT_ENUM_PRESET,
```

Comment:
`PYTHON_DEFAULT_ENUM_PRESET` is part of the default presets, which means it is already rendered 🙂

```suggestion
```

Reply:
done

Reply:
I added a test to cover proper enum generation. Looks fine :)